scripts/us_eia/opendata/process/test_data/nuc_status.mcf

```diff
@@ -0,0 +1,33 @@
+Node: dcid:Daily_Capacity_Nuclear_ForEnergyGeneration
+typeOf: dcid:StatisticalVariable
+name: "Nuclear Power Plant generating capacity"
+description: \""Nuclear generating capacity for a power plant. See https://www.eia.gov/nuclear/outages/.  For more information visit, Nuclear Regulatory Commission's Power Reactor Status Report\""
```

Comment:
hmm... should this be "\" ... \""?

Reply:
yea, wanted to check with you on it. is it only required for csv's? since this is in schema, maybe the escaping isn't necessary? it passes stat_var_checker though..

Reply:
Yeah only required for values that go into CSVs.